### ISSUE TITLE
NCA .nsp identified as CDN

### ISSUE THREAD

**Issue by nbalonso** [garoxas/NX_Game_Info#8]
Certain .nsp that have been clearly ripped because they are missing things like the game thumbnail when opened with XCI-Explorer, are being identified as CDN.

These same files are identified as NCA in other tools like Switch-Backup-Manager, which is in my opinion more accurate

**Comment by garoxas** [comment]
Can you share more details on which games are having this issue, like game titles and whether it's from certain scene release group

**Comment by garoxas** [comment]
After testing with several NSP files that show up as "NCA" in Switch Backup Manager, I found out that those files are missing ".cnmt.xml" file, but still have ".tik" and ".cert"
SBM expects those 3 files exist in order to flag them as NSP downloaded from CDN, but this app does differently. It only expects ".tik" and ".cert", because normally NSP converted from NCA will not have these files

The reason why XCI-Explorer does not work properly with these files is because it relies heavily on ".cnmt.xml", while this app (and SBM) will resort to use ".cnmt.nca" if that file is not present

tl;dr
Yes it's missing 1 file but it's not critical one. Same reason with why CDNSP ignored ".programinfo.xml" and ".nacp.xml"  in the first place

**Comment by garoxas** [comment]
FYI I've fixed XCI-Explorer so it will run properly on NSP file without cnmt.xml as it supposed to, just need to wait until it is merged in the main code
